### PR TITLE
Updated installing-vyper.rst with option to install Vyper by using a bash script

### PR DESCRIPTION
**Pull request by balajipachai**
### What I did
Created a bash script to install vyper on ubuntu systems.
Added those changes in the installing-vyper.rst

### How I did it
After following the **Installing Vyper** Section on **Read the Docs**, I thought about creating a bash script for installation purpose.
The script can be found at:
[Bash script to install vyper on Ubuntu](https://github.com/balajipachai/Scripts/blob/master/install_vyper/install_vyper_ubuntu.sh)

### How to verify it

> wget https://github.com/balajipachai/Scripts/blob/master/install_vyper/install_vyper_ubuntu.sh
> chmod 775 install_vyper_ubuntu.sh
> ./install_vyper_ubuntu.sh

### Description for the changelog
Added bash script to install vyper in installing-vyper.rst

### Cute Animal Picture
PR4

### CODE REVIEW

**Comment by fubuloubu** [comment]
I don't personally like linking to a 3rd party script with `sudo` in it.

Do you feel that the manual instructions unclear?

**Comment by balajipachai** [comment]
My point is either you install via the manual steps or just execute the bash script. 
If linking with a third party with sudo in it doesn't sounds a good idea, I have a suggestion we can create a scripts folder in the vyper codebase and under it this script will be stored and we can give this link in the docs.
In this way there will not be a third party linking and the script too resied in Vyper codebase and the community too can test it. 
Ultimately, the aim, is to get Vyper installed with as much ease as possible and using a bash script will help us in that

**Comment by fubuloubu** [comment]
I also make use of the python package directly. If you're used to virtual environments, then it's easier to do use `pip`. If not, I've found the the snap to be the easiest method, or Docker, which let's you install multiple versions easily.

**Comment by balajipachai** [comment]
> I also make use of the python package directly. If you're used to virtual environments, then it's easier to use `pip`. If not, I've found the the snap to be the easiest method, or Docker, which let's you install multiple versions easily.

I do understand, think, from a beginner's(newbies) perspective, who is keen on exploring Vyper and is unaware of virtual environments, pip, snap & docker, there executing the bash script and getting things installed will be quick and easy.

**Comment by yashvadhvani** [comment]
> > I also make use of the python package directly. If you're used to virtual environments, then it's easier to use `pip`. If not, I've found the the snap to be the easiest method, or Docker, which let's you install multiple versions easily.
> 
> I do understand, think, from a beginner's(newbies) perspective, who is keen on exploring Vyper and is unaware of virtual environments, pip, snap & docker, there executing the bash script and getting things installed will be quick and easy.

I agree on this as I am a newbie. It's tough to find things sometimes. At least dependencies should be installed without wasting much time.

**Comment by fubuloubu** [comment]
@jacqueswww Do you agree with my assessment above? Should we close this in favor of improving installation instructions for non-Linux systems?

**Comment by jacqueswww** [comment]
@balajipachai @fubuloubu please change this PR to just reference the above script and put under a section "Ubuntu LInux". I feel the script could still be useful for some, so they can check & download it themselves. Without having to explicitly instruct them to in the docs to download the script.

File: docs/installing-vyper.rst [hunk 1]
```diff
@@ -61,6 +61,14 @@ Run the following commands to install:
...
        sudo apt-get install python3-dev
 
+Using a BASH script
+^^^^^^^^^^^^^^^^^^^
+Vyper can be installed using a bash script.
+::
+
+    https://github.com/balajipachai/Scripts/blob/master/install_vyper/install_vyper_ubuntu.sh
+
+
```
Comment (fubuloubu) [inline docs/installing-vyper.rst]:
```suggestion

*Reminder*: Please read and understand the commands in any bash script before executing, especially with `sudo`.
```

**Pull request merged**